pkg/renderer/templates/public_values.yaml.tpl

```diff
@@ -165,6 +165,38 @@ scanner:
     {{- end }}
   {{- end }}
 
+scannerV4:
+  indexer:
+    {{- if .K8sConfig.ImageOverrides.Main }}
+    image:
+      {{- if .K8sConfig.ImageOverrides.Main.Name }}
```

Comment:
Does this matter? If so, then I think we'd need to update https://github.com/stackrox/stackrox/blob/master/operator/pkg/central/values/translation/image_overrides.go

Reply:
That's a good catch, but it might be more appropriate in [this PR](https://github.com/stackrox/stackrox/pull/8787) which adds scanner v4 support to the operator.

Reply:
This is a required change, in my opinion. I was testing this change locally, as described above. In particular I wanted to set specific image tags with `MAIN_IMAGE_TAG` and leverage the image tag overwriting machinery within our deployment scripts.

In the end the tag gets propagated to roxctl as something like this:

```
roxctl central generate k8s pvc -i quay.io/stackrox-io/main:4.3.x-555-gfoobar
```

Without this change to `pkg/renderer/templates/public_values.yaml.tpl` the main image tag would not end up in the scannerV4 configuration spec.
With this change the tag would be correctly injected there for scanner V4 as well.
[Note: Scanner V4 is build from the same tag as the main image.]

Reply:
I have been looking at the other file, @RTann, but I think this would definitely fall into the operator PR bucket (@johannes94). Otherwise this would end up being too much scope creep in this PR.

Reply:
@RTann can we resolve this?